lib/room.cpp

```diff
@@ -2456,8 +2456,8 @@ void Room::unban(const QString& userId)
 
 void Room::redactEvent(const QString& eventId, const QString& reason)
 {
-    connection()->callApi<RedactEventJob>(id(), QUrl::toPercentEncoding(eventId),
-                                          connection()->generateTxnId(), reason);
+    connection()->callApi<RedactEventJob>(id(), eventId),
```

Comment:
Typo in the previous suggestion
```suggestion
    connection()->callApi<RedactEventJob>(id(), eventId,
```